### ISSUE TITLE
awk: warning: escape sequence

### ISSUE THREAD

**Issue by cyayon** [brianreumere/gandi-automatic-dns#37]
Hi,

thanks for the 2.1.1 and fixed issue #36.

On archlinux, with GNU awk 5.3.0, there is a warning message repeated multiple times : 

```
awk: warning: escape sequence `\[' treated as plain `[
```

Is it possible to replace : 
```
field_name=$(printf "%s" "$i" | awk -F '": ?["\[]?' '{print $1}' | tr -d '"' | tr -d '[]')
field_value=$(printf "%s" "$i" | awk -F '": ?["\[]?' '{print $2}' | tr -d '"' | tr -d '[]')
```

with
```
field_name=$(printf "%s" "$i" | awk -F '": ?["[]?' '{print $1}' | tr -d '"' | tr -d '[]')
field_value=$(printf "%s" "$i" | awk -F '": ?["[]?' '{print $2}' | tr -d '"' | tr -d '[]')
```

thanks.

**Comment by brianreumere** [comment]
I like [your original suggestion to use `cut`](https://github.com/brianreumere/gandi-automatic-dns/issues/36#issuecomment-1868550705) so I went ahead and implemented that in [v2.1.2](https://github.com/brianreumere/gandi-automatic-dns/releases/tag/v2.1.2).

**Comment by cyayon** [comment]
Good ! I think it  also consumes less resources.
